modules/caddyhttp/fileserver/browse.go

```diff
@@ -46,12 +46,33 @@ import (
 //go:embed browse.html
 var BrowseTemplate string
 
+type ReturnType string
```

Comment:
I don't think we actually need this type, or the consts below, or the map -- why not just switch on the `Accept` header?

Reply:
I made the default return type a configurable option in Caddyfile and wanted to have some stronger typing and safeguards between those two modules. I don't like using "raw" strings and repeating them in multiple places but I can change the code if you prefer something simpler.